<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Groupbox */

QGroupBox {
    border: 1px solid #DDD;
    border-radius: 9px;
    margin-top: 9px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QGroupBox" name="OptionsGB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Options</string>
      </property>
      <widget class="QPushButton" name="LoadOBJ">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Load OBJ file</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="MeshGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>201</width>
         <height>111</height>
        </rect>
       </property>
       <widget class="QLabel" name="subDivisionSettingsTitleLabel">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>22</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Subdivision Settings</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="SubdivLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>75</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Subdivision steps:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="SubdivSteps">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>75</y>
          <width>30</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
       </widget>
      </widget>
      <widget class="QComboBox" name="MeshPresetComboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OpenCube</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Suzanne</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fertility</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Torus</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tetrahedron</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Octahedron</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Icosahedron</string>
        </property>
       </item>
      </widget>
      <widget class="QGroupBox" name="IsophotesGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>201</width>
         <height>191</height>
        </rect>
       </property>
       <widget class="QLabel" name="isophotesSettingsTitleLabel">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>22</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Isophotes Settings</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="colorStripesLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Color of stripes</string>
        </property>
       </widget>
       <widget class="QLabel" name="frequencyLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Frequency of stripes:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="frequencySteps">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>60</y>
          <width>32</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
       <widget class="QComboBox" name="colorStripesComboBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Black &amp; White</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red &amp; White</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blue &amp; White</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QGroupBox" name="RendererGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>201</width>
         <height>131</height>
        </rect>
       </property>
       <widget class="QLabel" name="renderSettingsTitleLabel">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>22</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Render Settings</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QCheckBox" name="isophotesCheckBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Isophotes</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="phongShadingCheckBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Phong shading</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="MainView" name="MainDisplay">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainView</class>
   <extends>QOpenGLWidget</extends>
   <header>mainview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
